src/src/PluginStructs/P116_data_struct.cpp

```diff
@@ -407,10 +407,10 @@ bool P116_data_struct::plugin_write(struct EventStruct *event, const String& str
 /****************************************************************************
  * displayOnOff: Turn display on or off
  ***************************************************************************/
-void P116_data_struct::displayOnOff(bool    state,
-                                    int8_t  backlightPin,
-                                    uint8_t backlightPercentage,
-                                    uint8_t displayTimeout) {
+void P116_data_struct::displayOnOff(bool     state,
+                                    int8_t   backlightPin,
```

Comment:
Why aren't these stored values a member of this datastruct?
Now you can only call these functions from the .ino file, with the event present.
Also you need to include them over and over again, while they are known during `PLUGIN_INIT` and thus don't change during the lifetime of the P116_data_struct object.

Reply:
Hmm, I don't remember the exact reason, but it looks like that should be improved. I'll fix that.